app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "../views/invites/render_json"
+
+class InvitesController < ApplicationController
+  include InvitesJson
+
+  def index
+    user = @current_user
+    render status: 200,
+      json: groups_invited_to(user)
+  end
+
+  # POST request for inviting another user to a group.
+  def create
```

Comment:
suppose the user has 6 group memberships, 4 of which are invites (because ```accepted: false```) then the params actually need to be something like 
```
{
    invites: {
        invite: {
            invite_id: first_invite = user.group_memberships.where(accepted: false).first
            user_id: first_invite.user_id
            group_id: first_invite.group_id
       }
       invite {
            invite_id: second_invite = user.group_memberships.where(accepted: false).second
            user_id: second_invite.user_id
            group_id: second_invite.group_id
    }
    ...
}```

or something which i know isn't even close to being a valid thing...
@lpil  help?